styles/desktop.css

```diff
@@ -1,4 +1,4 @@
-@media (min-width: 768px) {
+@media (min-width: 769px) {
```

Comment:
- [ ] Great job on your desktop design! It's really coming together nicely. I did notice **a small issue with responsiveness,** though. When I adjusted the resolution, I noticed that your design breaks around 800px. No worries, though! I have a friendly suggestion for you to try. **Instead of using fixed widths, you could experiment with using percentages.** This way, your design will adapt better to different screen sizes.

   Additionally, here's a handy tip that might help you identify which elements are causing the design to break. You can apply this class to your CSS:

   ```
   * {
     border: 1px solid salmon;
   }
   ```
  By doing this, you'll be able to see which elements are going outside of their intended boundaries and affecting your design. Good luck, and keep up the great work!

   **_Video for a better understanding:_**
   
   [scrnli_7_12_2023_10-50-02 AM.webm](https://github.com/Chimoneg27/Personal-Portfolio/assets/107952724/8e92d2f5-8281-4b2b-a727-58b5bd2317b4)